http_logs/challenges/default.json

```diff
@@ -121,6 +121,22 @@
           "warmup-iterations": 200,
           "iterations": 100,
           "target-throughput": 0.5
+        },
+        {
```

Comment:
What is the intention of adding reindex to the default `append-no-conflicts` challenge? Do you want to get some sort of graph displayed in https://elasticsearch-benchmarks.elastic.co/#tracks/http-logs/nightly/default/30d and if yes what metrics?

Reply:
Good question. I think it could be either docs/s or total run-time (docs/s is probably better, but harder to find)? What will I need to do in order to get it to show up?

Reply:
To get metrics (depending on what the `_reindex` API returns) you should stick with the runner that you've created and then return a tuple or dict as described [here](https://esrally.readthedocs.io/en/stable/adding_tracks.html#custom-runners) -- under `This function can return:` -- where you can define `weight/unit`. You can additionally specify more KV pairs that will be appended in the meta section of the metric records.

Reply:
I removed the addition to the default challenge and added docs/s metric for reindex in c86dcc2ee620ed2b6e06e45da817aa69bb5f496e